src/test/kotlin/com/exactpro/epfast/template/dsl/builders/FieldBuilder.kt

```diff
@@ -0,0 +1,29 @@
+package com.exactpro.epfast.template.dsl.builders
+
+import com.exactpro.epfast.template.simple.FieldInstruction
+import kotlin.reflect.KProperty
+
+fun <T> javaProperty(getter: () -> T, setter: (T) -> Unit) = JavaProperty(getter, setter)
```

Comment:
You use these methods in other files as well. They deserve to be placed in separate file